.github/workflows/markdown-link-check-config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
For this to be a drop in replacement, we need to only check changed files with `check-modified-files-only` option.

Reply:
We only wrote that functionality into the markdown checker because it was breaking on certain sites, and we knew the file it was breaking on wasn’t modified often. I think it’s safe to check all top-level files again.

Reply:
Also we need to check all links on each PR because a linked file could be moved, breaking links in unmodified docs.

Reply:
If this will no longer check `design`, `proposals`, thats fine with me since those need to be moved anyway.

Reply:
Cool, since we will be checking many fewer links, hopefully we won't get the 429s from github.